plugins/kafka/kafka.cpp

```diff
@@ -54,24 +58,80 @@ namespace KafkaPlugin
     // Static Methods (internal)
     //--------------------------------------------------------------------------
 
+#ifdef _CONTAINERIZED
+    static void applyConfigProps(IPropertyTree * props, const char* configName, RdKafka::Conf* configPtr)
+    {
+        if (props)
+        {
+            Owned<IPropertyTreeIterator> iter = props->getElements(configName);
+            std::string errStr;
+
+            ForEach(*iter)
+            {
+                IPropertyTree & entry = iter->query();
+                const char* name = entry.queryProp("@name");
+                const char* value = entry.queryProp("@value");
+
+                if (name && *name)
+                {
+                    if (stricmp(name, "metadata.broker.list") != 0)
```

Comment:
trivial: clearer to combine the 2 ifs statements and use strsame

Reply:
Agreed, but the else from the first if() leads to a DBGLOG that is useful. I did change the second if() to use !strisame() though. Also changed similar test on line ~146.

Reply:
DBGLOG may be enough, but at some point we aim to turn them off, so should they be OWARN or similar? (or will that be too verbose

Reply:
Excellent point. I converted a few DBGLOG calls to OWARNLOG.